apps/strapi-cms/.env.example

```
@@ -6,6 +6,14 @@ ADMIN_JWT_SECRET=changeme
 TRANSFER_TOKEN_SALT=changeme
 JWT_SECRET=changeme
 
+#AWS S3 CONFIG
+AWS_PROVIDER=local
```

In my opinion, the name should be provider-agnostic, that's why I would rename it to `UPLOAD_PLUGIN_PROVIDER`.

```suggestion
UPLOAD_PLUGIN_PROVIDER=local
```